src/css/styles/_c-bullet-point.scss

```diff
@@ -0,0 +1,183 @@
+.c-bullet-point {
```

Comment:
In my opinion we mix component styling with layout.
`c-bullet-point` should be a component without any layout info.

Title, divider, description and even bullet point should be style without absolute and relative.

![image](https://user-images.githubusercontent.com/789841/50595633-870fe480-0ea1-11e9-9232-10f94b726f75.png)

⚠️  Note: only draft

```diff
diff --git a/src/css/styles/_c-bullet-point.scss b/src/css/styles/_c-bullet-point.scss
index 7d2ab94..9697baa 100644
--- a/src/css/styles/_c-bullet-point.scss
+++ b/src/css/styles/_c-bullet-point.scss
@@ -32,7 +32,6 @@
   }
 
   &__title {
-    position: absolute;
     bottom: 0.8rem;
     font-size: 1.2rem;
     font-weight: bold;
@@ -45,18 +44,18 @@
   }
 
   &__divider {
-    position: absolute;
     width: 2rem;
     height: 3px;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
     background-color: $col-red;
 
     &--right {
-      right: 0;
+      float: right;
     }
   }
 
   &__description {
-    position: absolute;
     top: 1rem;
     font-size: 1rem;
 
diff --git a/src/css/styles/_general.scss b/src/css/styles/_general.scss
index a4bf8fe..6f3b1ea 100644
--- a/src/css/styles/_general.scss
+++ b/src/css/styles/_general.scss
@@ -4,6 +4,7 @@
   box-sizing: border-box;
   padding: 0;
   margin: 0;
+  clear: both;
 }
 
 .app {

```

Reply:
Hmm, ok, I think title, divider and description can be handled without absolute positioning. Absolute positioning seemed to be a better choice for control and precision, but I get your point of view- now it's done without absolute positioning. 
But still- bullets seem to be more naturally positioned, if done by absolute.